test/missing_semicolon.html

```diff
@@ -21,4 +21,4 @@
 <body>
 
 </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
Can we please remove this unrelated change so that the changes are logged only in the relevant lines? Thanks!